### DISCUSSION TITLE
过期：懒人包&简易包 2022.1011-1014

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#195] [回收]
### 懒人包
当前版本存在的可改进问题 https://github.com/hooke007/MPV_lazy/issues/196
所涉脚本已模块化，如需更新需要先移除旧脚本，再添加整个 uosc 文件夹

### 简易包
初版存在错误，已重传修复。

<hr />
链接 <a href='https://github.com/hooke007/MPV_lazy/releases/tag/20221011'>mpv-lazy-20221011</a>

链接 <a href='https://github.com/hooke007/mpv.net_CM/releases/tag/6032'>mpvnet-CM-20221014</a>

**Comment by ic2391** [comment]
![image](https://user-images.githubusercontent.com/62416821/195335925-6473308c-fd67-476b-be76-d2f7284ec939.png)
版主好，请问设置切换列表文件快捷键不起作用，是哪里没设置好吗？

**Reply by hooke007** [reply]
确保你的列表有文件。你看到的是文件浏览器还是播放列表？

**Reply by ic2391** [reply]
![image](https://user-images.githubusercontent.com/62416821/195349788-e8667ad3-0e9f-4f09-9d43-febae09ee5a3.png)
我不太清楚是文件浏览器还是播放列表，播放器界面里 上一个下一个 有效，快捷键无效

**Reply by hooke007** [reply]
你没拉多个文件进播放器就自然列表没有其他文件。

功能不同，那是切换下一个文件浏览器的中的文件（如果列表没有文件）。
中间那个按钮<kbd>📋</kbd>会显示你的列表存不存在其他条目，有就显示列表，没有就显示浏览器

**Reply by hooke007** [reply]
如果需要类似autoload的功能，参考uosc的配置文件开启。

此外 <kbd>></kbd>这个按钮的对应的指令是 `script-binding uosc/next` ,也可以通过重绑定实现类似表现

**Reply by ic2391** [reply]
谢谢，就是需要autoload的功能，弄好了

**Reply by hooke007** [reply]
@yubanmeiqin9048 不要在非关联帖发无关问题，你的需求参考此处

**Comment by zxzxm** [comment]
rife补帧失败，求解答
![Snipaste_2022-10-13_01-22-41](https://user-images.githubusercontent.com/3539911/195408825-9b1c4d67-cf83-47ea-964a-928b1999aff4.png)

**Reply by hooke007** [reply]
https://github.com/HomeOfVapourSynthEvolution/VapourSynth-RIFE-ncnn-Vulkan/issues/40

**Reply by zxzxm** [reply]
只有intel集显，我重装一下显卡驱动试试。
![Snipaste_2022-10-13_09-19-11](https://user-images.githubusercontent.com/3539911/195476965-eabd8bda-8bcc-4be9-bf19-e761f57bb0b1.png)

**Reply by hooke007** [reply]
我的意思是驱动的问题。不是品牌的问题
就算被迫改成1，速度就太慢了没法实用了

**Reply by hooke007** [reply]
这显卡的性能什么水平。。。没有2060mobile的话就放弃rife吧，一般核显折腾也没意义，只能跑个位数

**Reply by zxzxm** [reply]
了解了 谢谢

**Reply by hooke007** [reply]
obs版共存了三种补帧，我的预测是svpflow最适合你的机器。

**Comment by linger040831** [comment]
请问https://github.com/hooke007/MPV_lazy/releases/tag/20221011 这个版本自动播放下一集需要改哪里的参数，是要autoload这个脚本嘛？我研究了好久没研究明白，麻烦老哥能帮我一下😭

**Reply by hooke007** [reply]
一楼不就是？

**Reply by linger040831** [reply]
https://raw.githubusercontent.com/hooke007/MPV_lazy/main/portable_config/script-opts/uosc.conf 默认的包里为啥不带这文件啊，还要手动创建😂

**Reply by hooke007** [reply]
你猜为什么有个文件叫script-opts.conf

**Reply by linger040831** [reply]
script-opts-append = uosc-autoload=yes
请问这么写，就可以不需要uosc.conf这个文件吗？

**Reply by hooke007** [reply]
更新记录里有写为什么这么做。

仓库也有对应的参考文件。

**Reply by linger040831** [reply]
好的，谢谢您

**Comment by zhongfly** [comment]
想问一下 .conf的格式化是用什么工具或者编辑器插件做的？

**Reply by hooke007** [reply]
是指什么格式化？

**Reply by zhongfly** [reply]
就是配置文件里的=对齐
```
 d3d11-exclusive-fs = no
 d3d11-flip         = yes 
```
而不会是：
```
 d3d11-exclusive-fs = no
 d3d11-flip = yes 
```

**Reply by hooke007** [reply]
就是手动对的。。。

**Reply by zhongfly** [reply]
> 就是手动对的。。。

啊……我还以为有什么工具可以，我感觉手工对还挺麻烦的，所以觉得不太可能

**Reply by hooke007** [reply]
一开始的时候mpv.conf用参数并不多，后边是一条一条加的，所以没有感觉工作量大

**Reply by hooke007** [reply]
我见过有人用TAB做对齐的，也是一行行做，比我敲空格略快点
![Snipaste_2022-11-04_00-51-35](https://user-images.githubusercontent.com/41094733/199861976-8d86ce50-0be0-4dba-a11c-5399c03f90e3.jpg)

**Reply by hooke007** [reply]
我看了一下vsc有个插件好像能用 https://marketplace.visualstudio.com/items?itemName=wwm.better-align

**Reply by zhongfly** [reply]
> 我看了一下vsc有个插件好像能用 https://marketplace.visualstudio.com/items?itemName=wwm.better-align

这个对目前的mpv.conf用不了，只能处理相连行的，如果中间有单独的注释行隔开，就不会对齐到同一个=

我觉得可以试试https://marketplace.visualstudio.com/items?itemName=yo1dog.cursor-align 和 https://marketplace.visualstudio.com/items?itemName=jca02266.align-columns
按住alt，鼠标点击（多选）要对齐的地方，然后 前者alt+a进行光标对齐 / 后者crtl+p唤出命令面板选择 Align multi-cursors

**Comment by suzakuiname** [comment]
大佬我打开视频有坏点马赛克和卡顿现象 用电脑自带播放器没有 请问是什么原因

**Reply by hooke007** [reply]
这种没有样本、没有日志、也没有详细描述的问题我没法回答

**Reply by suzakuiname** [reply]
用mpvnet得到解决 还不知道原因
